mne/time_frequency/multitaper.py

```diff
@@ -331,9 +331,9 @@ def psd_array_multitaper(x, sfreq, fmin=0.0, fmax=np.inf, bandwidth=None,
         The sampling frequency.
     %(fmin_fmax_psd)s
     bandwidth : float
-        Half-bandwidth of the multi-taper window function in Hz. For a given
-        frequency, frequencies at ± half-bandwidth are smoothed together.
-        The default value is a half-bandwidth of 4.
+        Frequency bandwidth of the multi-taper window function in Hz. For a
+        given frequency, frequencies at ± bandwidth/2 are smoothed together.
+        The default value is a bandwidth of 8 * (sfreq / n_times).
```

Comment:
```suggestion
        given frequency, frequencies at ``± bandwidth / 2`` are smoothed together.
        The default value is a bandwidth of ``8 * (sfreq / n_times)``.
```

Thanks for catching this one! I did not look at the code in detail when I edited those docstrings..